src/core/sequence_batch_scheduler.cc

```diff
@@ -513,6 +513,9 @@ SequenceBatchScheduler::Enqueue(std::unique_ptr<InferenceRequest>& irequest)
       irequest->Trace(), TRITONSERVER_TRACE_QUEUE_START,
       irequest->QueueStartNs());
 
+  // Record time at the beginning of the batcher queueing
+  irequest->CaptureBatcherStartNs();
```

Comment:
I guess it is ok, and expected, that the dynaseqbatcher will overwrite this for oldest? You may want to add something in the comment about that.

Reply:
Yes, batcher start time should be overwritten for the use of current batcher, as the previous batcher has finished its scheduling of the request by this point. Added comment